server/test/com/cloud/resource/MockResourceManagerImpl.java

```diff
@@ -172,6 +173,12 @@ public Cluster getCluster(final Long clusterId) {
         return null;
     }
 
+    @Override
+    public DataCenter getZone(Long zoneId) {
+        // TODO Auto-generated method stub
```

Comment:
Because we have tools to search for TODOs, please remove the comment if there is nothing outstanding to be done.